app/views/users/_header.html.erb

```diff
@@ -1,10 +1,9 @@
 <!--Descriptive page name, messages and instructions-->
 <div class="primary header module">
-	<% 	display_name = @pseud ? @pseud.name : @user.login	%>
-	<h2 class="heading">
-	  <%= @pseud ? link_to_unless_current(display_name, user_pseud_path(@user, @pseud)) : link_to_unless_current(display_name, @user) %>
```

Comment:
Just noting that it looks like this was only ever _actually_ displayed as a link on a user's profile page, so I don't think `link_to_unless_current` offered much value, considering the "Dashboard" link it right there.

We could go back to using a link if other people think it's helpful, though.